config/jobs/ppc64le-cloud/jenkins-infra/jenkins-infra-postsummit.yaml

```diff
@@ -0,0 +1,23 @@
+postsubmits:
+  ppc64le-cloud/jenkins-infra:
+    - name: load-jenkins-infra-job
+      branches:
+      - master
+      decorate: true
+      spec:
+        containers:
+          - image: quay.io/powercloud/jenkins-infra:0.1
+            envFrom:
+            - secretRef:
+                name: jenkins-secret
+            command:
+              - sh
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -x
+                export JENKINS_URI="https://jenkins.ppc64le-cloud.org"
+                chmod +x ./hack/load-all-job.sh
```

Comment:
see if you can set the right permission in the repo `ppc64le-cloud/jenkins-infra` itself.

Reply:
@mkumatag Removed setting permission